.travis.yml

```diff
@@ -31,6 +31,22 @@ env:
 #   https://docs.travis-ci.com/user/osx-ci-environment/
 matrix:
   include:
+    - os: windows
+      language: shell
+      before_install:
+        - choco install python2 -x86
```

Comment:
`choco install python2 -x86` installs `32 bit libs` in a `32 bit` system.
without `-x86` it installs `64 bit libs ` also which causes error during configuration.